semantic/semantic.fbs

```diff
@@ -1,5 +1,14 @@
 namespace fbsemantic;
 
+table TypeEnvironment {
```

Comment:
The schema has a `root_type` declaration at the bottom:
```
root_type Package;
```
I'm not sure if it will be possible to create a flatbuffer that has something else at the root.  I'm curious if you've tried this?

Reply:
Yes it's possible via the `get_root` function:

    let buf = include_bytes!(concat!(env!("OUT_DIR"), "/prelude.data"));
    flatbuffers::get_root::<fb::TypeEnvironment>(buf).into()

That's why we can serialize and deserialize `PolyType`s independent of the semantic node types.